src/mlagility/cli/cli.py

```diff
@@ -131,6 +132,14 @@ def main():
         default=filesystem.DEFAULT_CACHE_DIR,
     )
 
+    benchmark_parser.add_argument(
+        "--label",
```

Comment:
Should this be `--labels` since its a list? the `dest` is even `labels`.

Fix in the docs as well if you decide to change it.

Reply:
My vote is for it to be `--labels`, especially since the `benchmark_scripts()` API argument is `labels`

Reply:
Made the same comment above before seeing this, vote +1.